packages/kit/src/exports/index.js

```diff
@@ -1,4 +1,4 @@
-import { HttpError, Redirect } from '../runtime/control.js';
+import { HttpError, Redirect, ValidationError } from '../runtime/control.js';
```

Comment:
One minor request, based on a couple of weeks working with the new helpers: can we make the status code the final argument, with a sensible default? So... 

```ts
export const redirect = (location: string, status = 303) => {...}; // not quite sure what'd be  the correct default
export const error = (message: string, status = 500) => {...};
export const invalid<T> = (data: T, status = 400) => {...};
```
If not, fine.

Reply:
Prettier will butcher `invalid(data, status)`:

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/1162160/188498574-c4fc81bc-c63e-4046-b91a-7b40bec54f6b.png">

It won't do that for strings, but in the interests of consistency (and also encouraging people not to blindly use the defaults — and as you say, there isn't a plausible default for `redirect`) it's better to have the status code first

Reply:
I prefer the leading status code because it clearly communicates the intent. Having a "sane" default might also get on your way. "Explicitly over magic" or something along the lines. Sorry for butchering the messages I picked up in your comments, Rich. ;-)

Each response has to have a status code, not a body or message though. Having an optional parameter forces it to the tail of the parameter list, which closes the API to future changes/improvements.

Also, I usually don't care too much about the message if the status code clearly communicates what's going on. 

Just my two cents, thanks for listening to my Ted talk.

Reply:
Ok. Got it. Valid points. Yet another chapter ripped out of  "Chris Carson's Big Book of Code Style, As Divinely Revealed to Chris Carson by Chris Carson" and used to light the barbecue, but I'll survive.